README.md

```diff
@@ -22,9 +22,17 @@ In the specifications in the above table of contents, the keywords "MUST", "MUST
 
 To provide context for users the following section gives example use cases for each part of the spec.
 
-## Filesystem Bundle & Configuration
+**Container Configuration**
 
-- A user can create a root filesystem and configuration, with low-level OS and host specific details, and launch it as a container under an Open Container runtime.
+User: Application writer. Container configuration defines the environment that an application can rely on being available when it is launched inside of a container, this includes things such as environment variables, a working directory, mount points, and operating system permissions.
```

Comment:
I'd use “bundle author” or maybe “packager” here instead of “application writer”, since the later might also sounds like an author of the bundled software (e.g. [Nginx authors][1]), when we mean the one who put the bundle together and wrote the OCI config(s).  And there's also some [dislike of the word “application”][2] (although I'm personally fine with it).

[1]: https://github.com/nginx/nginx/graphs/contributors
[2]: https://github.com/opencontainers/specs/pull/107#issuecomment-145615479